ci/aws-analytical-env/shared/meta.yml

```diff
@@ -471,8 +471,65 @@ meta:
             args:
               - -exc
               - |
-                AWS_INGESTION_INPUT_STORAGE_BUCKET="$(cat terraform-output-common-crown/outputs.json |  jq -r '.ingestion_input_storage.value.id')"
-                aws s3 --endpoint-url=https://s3-eu-west-1.amazonaws.com sync cyi-files/ s3://${AWS_INGESTION_INPUT_STORAGE_BUCKET}/${S3_FOLDER}
+                # use 25... ingestion-input-storage bucketID
+                AWS_PUBLISHED_BUCKET="SOME-BUCKET-ID"
```

Comment:
Should this not be set to some actual bucket or default to something meaningful?

Reply:
the `output.json` doesn't have bucket ID for the target bucket and since its temp for Crown Concourse, I have `avaiator`d the id manually and kept the code with comment/placeholder.